taqueria-lib-ligo/common.ts

```diff
@@ -1,79 +1,131 @@
-import { getDockerImage, sendErr } from '@taqueria/node-sdk';
-import { ProxyTaskArgs, RequestArgs } from '@taqueria/node-sdk/types';
-import { join } from 'path';
+import { getDockerImage, sendErr } from "@taqueria/node-sdk";
+import { ProxyTaskArgs, RequestArgs } from "@taqueria/node-sdk/types";
+import * as fs from "fs";
+import { delimiter, join } from "path";
 
 export interface LigoOpts extends ProxyTaskArgs.t {
-	command: string;
+  command: string;
```

Comment:
I'm not sure why the formatter is inconsistent here. Made sure I ran `dprint` finishing up